sdk/ai/Azure.AI.Agents.Persistent/src/Azure.AI.Agents.Persistent.csproj

```diff
@@ -16,7 +16,6 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Core" />
-    <PackageReference Include="Azure.Identity" />
     <PackageReference Include="System.ClientModel" />
```

Comment:
Please remove this as well.   You've got a reference to client model through Core and shouldn't be taking a direct as well.